src/work-allocation/components/priority-field/priority-field.component.ts

```diff
@@ -11,17 +10,34 @@ export class PriorityFieldComponent {
   /**
    * The current date being examined
    */
-  @Input() public dueDate: Date;
+  @Input() public date: Date;
+
+  @Input() public majorPriority?: number;
 
   public get priority(): TaskPriority {
-    const today = new Date();
-    if (this.dueDate) {
-      const isToday = this.dueDate.toDateString() === today.toDateString();
-      if (isToday) {
+
+    const dCount = this.daysBetween(this.date, new Date());
```

Comment:
This OK for now, but we need to change it so that it handles the time part of the due date, which is currently ignored. There will be separate tickets for this